src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {person.name}
```

Comment:
```suggestion
        {`My name is ${person.name}`}
```

Next time pls use template strings